net/crowdsec/Makefile

```
@@ -0,0 +1,189 @@
+  community-wide IP reputation database 
+endef
+
+define Package/crowdsec/description
+$(call Package/crowdsec/Default/description)
+
+  This package contains the main program.
+endef
+
+define Package/golang-crowdsec-dev/description
+$(call Package/crowdsec/Default/description)
+
+  This package provides the source files for the program.
+endef
+
+define Package/crowdsec/install
+	$(call GoPackage/Package/Install/Bin,$(1))
+
+	$(INSTALL_DIR) $(1)/etc/crowdsec
+	$(INSTALL_DIR) $(1)/etc/crowdsec/scenarios
+	$(INSTALL_DIR) $(1)/etc/crowdsec/postoverflows
+	$(INSTALL_DIR) $(1)/etc/crowdsec/collections
+	$(INSTALL_DIR) $(1)/etc/crowdsec/patterns
+	$(INSTALL_DIR) $(1)/etc/crowdsec/hub
+	
+	$(INSTALL_DATA) \
+		$(GO_PKG_BUILD_DIR)/src/$(GO_PKG)/config/config.yaml \
+		$(1)/etc/crowdsec
+	$(INSTALL_DATA) \
+		$(GO_PKG_BUILD_DIR)/src/$(GO_PKG)/config/dev.yaml \
+		$(1)/etc/crowdsec
+	$(INSTALL_DATA) \
+		$(GO_PKG_BUILD_DIR)/src/$(GO_PKG)/config/user.yaml \
+		$(1)/etc/crowdsec
+	$(INSTALL_DATA) \
+		$(GO_PKG_BUILD_DIR)/src/$(GO_PKG)/config/acquis.yaml \
+		$(1)/etc/crowdsec
+	$(INSTALL_DATA) \
+		$(GO_PKG_BUILD_DIR)/src/$(GO_PKG)/config/profiles.yaml \
+		$(1)/etc/crowdsec
+	$(INSTALL_DATA) \
+		$(GO_PKG_BUILD_DIR)/src/$(GO_PKG)/config/simulation.yaml \
+		$(1)/etc/crowdsec
+	$(INSTALL_DATA) \
+		$(GO_PKG_BUILD_DIR)/src/$(GO_PKG)/config/local_api_credentials.yaml \
+		$(1)/etc/crowdsec
+	$(INSTALL_DATA) \
+		$(GO_PKG_BUILD_DIR)/src/$(GO_PKG)/config/online_api_credentials.yaml \
+		$(1)/etc/crowdsec
+
+	$(CP) \
+		$(GO_PKG_BUILD_DIR)/src/$(GO_PKG)/config/patterns/* \
+		$(1)/etc/crowdsec/patterns
+
+	$(INSTALL_DIR) $(1)/srv/crowdsec/data/
+	
+	$(INSTALL_DIR) $(1)/etc/init.d
+	$(INSTALL_BIN) \
+		./files/crowdsec.initd \
+		$(1)/etc/init.d/crowdsec
+		
+	$(INSTALL_DIR) $(1)/etc/config
+	$(INSTALL_CONF) \
+		./files/crowdsec.config \
+		$(1)/etc/config/crowdsec
+endef
+
+define Package/crowdsec/postinst
+#!/bin/sh
+if [ -z "$${IPKG_INSTROOT}" ]; then
+  /etc/init.d/crowdsec enable > /dev/null 2>&1
+  /etc/init.d/crowdsec restart > /dev/null 2>&1
+  ln -s /usr/bin/crowdsec-cli /usr/bin/cscli > /dev/null 2>&1
+  cscli -c /etc/crowdsec/config.yaml machines add --auto > /dev/null 2>&1
+  cscli -c /etc/crowdsec/config.yaml capi register > /dev/null 2>&1
+  cscli hub update > /dev/null 2>&1
+  cscli parsers install "crowdsecurity/whitelists" > /dev/null 2>&1
+  /etc/init.d/crowdsec restart > /dev/null 2>&1
+fi
+endef
+
+define Package/crowdsec/prerm
+#!/bin/sh
+if [ -z "$${IPKG_INSTROOT}" ]; then
+  /etc/init.d/crowdsec stop > /dev/null 2>&1
+  /etc/init.d/crowdsec disable > /dev/null 2>&1
+  rm -f /usr/bin/crowdsec > /dev/null 2>&1
+  rm -f /usr/bin/cscli > /dev/null 2>&1
+  rm -f /usr/bin/crowdsec-cli > /dev/null 2>&1
+  rm -f /etc/init.d/crowdsec > /dev/null 2>&1
+fi
+endef
+
+define Package/crowdsec/conffiles
+  /etc/crowdsec/config/config.yaml
+  /etc/crowdsec/config/local_api_credentials.yaml
+  /etc/config/crowdsec
```

```suggestion
/etc/crowdsec/config/config.yaml
/etc/crowdsec/config/local_api_credentials.yaml
/etc/config/crowdsec
```

Should all of `/etc/crowdsec` be saved?

---

fixed,
thanks

---

/etc/crowdsec folder is not saved, it is /etc/config/crowdsec file